### PR TITLE
Add all browser bundles (for master)

### LINKED ISSUE #53: Automatically build and host UMD package when publishing on npm
This seems really cool: https://unpkg.com/#/

see: https://unpkg.com/bigchaindb-driver@0.1.0/dist/browser/

### PR DESCRIPTION
**Pull request by TimDaub**
Fixes #53 and #24

#55 is against current release branch. This is against master.

I'm not sure how to handle such things :confused: 

I defer to @vrde

Edit:

todo:

- [x] Fix `browser` value in package.json

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -14,6 +14,39 @@ const PATHS = {
     NODE_MODULES: path.resolve(__dirname, 'node_modules'),
 }
 
+const OUTPUTS = [
```
Comment (diminator) [inline webpack.config.js]: not sure if we want to use `bundle.*`?
This would become unclear in local imports
propsal: `bigchaindb-driver.* `

Comment (TimDaub) [inline webpack.config.js]: ok, i'll change that.

Comment (TimDaub) [inline webpack.config.js]: changed.

**Comment by TimDaub** [comment]
This is me currently: http://classicprogrammerpaintings.com/post/142586036029/junior-programmer-learns-git-rebase

File: README.md [hunk 2]
```diff
@@ -17,13 +17,14 @@
...
-* [Installation](#installation)
+* [Node.js Installation and Usage](#node.js-installation-and-usage)
    * [Example: Create a transaction](#example-create-a-transaction)
+* [Browser Installation and Usage](#browser-installation-and-usage)
 * [Documentation](#bigchaindb-documentation)
 * [Authors](#authors)
 * [License](#license)
 
-## Installation
+## Node.js Installation and Usage
```
Comment (vrde) [inline README.md]: This is true also for browsers. I'd rename this section to something like `Installation and Usage with package managers (npm/yarn)`.

File: README.md [hunk 3]
```diff
@@ -74,6 +75,65 @@ conn.postTransaction(txSigned)
     .then((res) => console.log('Transaction status:', res.status))
 ```
 
+## Browser Installation and Usage
```
Comment (vrde) [inline README.md]: I'd rename this to something like `Use a pre-built image (browser only)`.

**Pull request merged**